packages/store/src/services/backend-system.ts

```diff
@@ -81,5 +90,31 @@ export class SystemService implements Service<BackendSystem, BackendSystemKey> {
 }
 
 export function getInstance(logger: Logger, options: ServiceOptions = {}): SystemService {
+    if (!options.baseDirectory) {
+        ensureSettingsMigrated();
+        options.baseDirectory = getSapDevToolsDirectory();
+    }
     return new SystemService(logger, options);
 }
+
+/**
+ * Ensure settings are migrated from the old fiori tools directory to the new sap development tools directory.
+ */
+function ensureSettingsMigrated(): void {
+    const sapDevToolsDir = getSapDevToolsDirectory();
+    const migrationFlag = join(sapDevToolsDir, '.migrated');
```

Comment:
```suggestion
    const migrationFlag = join(sapDevToolsDir, '.systemsMigrated');
```
Can we make this more specific?